modin/pandas/general.py

```diff
@@ -51,41 +51,82 @@ def notna(obj):
 def merge(
     left,
     right,
-    how: str = "inner",
+    how="inner",
```

Comment:
Why were the type hints removed?

Reply:
Ideally all of the code will have type hints. It is better for the developer and data scientist to understand what can be passed for parameters.

Reply:
I see. Just need to think on that we start to add type hints.

Reply:
Left as it was.